common/app/model/meta.scala

```diff
@@ -427,7 +427,7 @@ case class StoryQuestionsAtomPage(
   override val withVerticalScrollbar: Boolean
 )(implicit request: RequestHeader) extends AtomPage {
   override val atomType = "storyquestions"
-  override val body = views.html.fragments.atoms.storyquestions(atom, isAmp = false)
+  override val body = views.html.fragments.atoms.storyquestions(atom, isAmp = false, isDemandTest = false)
```

Comment:
I missed this line. By removing the additional parameter, you would include apps users into the test as well. If that is a good thing, I would advise moving the switch inside the template as suggested. If not, then by all means ignore my comment.